tower-http/src/trace/on_failure.rs

```diff
@@ -10,20 +10,23 @@ pub trait OnFailure<FailureClass> {
     /// Do the thing.
     ///
     /// `latency` is the duration since the request was received.
-    fn on_failure(&mut self, failure_classification: FailureClass, latency: Duration);
+    ///
+    /// `span` can be used to record field values that weren't know when the span was
+    /// created.
```

Comment:
again, maybe:
```suggestion
    /// `span` is the `tracing` [`Span`] corresponding to this request, produced
    /// the closure passed to [`TraceLayer::make_span_with`]. It can be used to
    /// [record field values][record] that weren't known when the span was
    /// created.
    ///
    /// [`Span`]: https://docs.rs/tracing/latest/tracing/span/index.html
    /// [record]: https://docs.rs/tracing/latest/tracing/span/struct.Span.html#method.record
```